<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sleepDialog</class>
 <widget class="QDialog" name="sleepDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>160</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>Sleep</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>128</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="sleepOff">
       <property name="text">
        <string>Sleep Off</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sleepOn">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sleepValue1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Current value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sleepEdit">
       <property name="toolTip">
        <string>Sleep timer start value in milliseconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>160</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>Screensaver</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>128</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="screenOff">
       <property name="text">
        <string>Screensaver off</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="screenOn">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="screenValue1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Current value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="screenEdit">
       <property name="toolTip">
        <string>Screensaver start time in milliseconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>231</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Sleep / Screensaver timer adjustment</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="deviceLabel">
      <property name="text">
       <string>devicename here</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>sleepDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>sleepDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
